HTML5-frontend/ndt-wrapper.js

```diff
@@ -122,8 +117,14 @@ NDTWrapper.prototype.run_test = function () {
         }
       };
 
-      this.client = new NDTjs(this._hostname, this._port, this._path, this.callbacks, this._update_interval);
-      this.client.startTest();
+      if (this._hostname) {
+        this.client = new NDTjs(this._hostname, this._port, this._path,
+                                this.callbacks, this._update_interval);
+        this.client.startTest();
+      } else {
+        console.log('Could not determine a suitable NDT server. Aborting.');
+        return false;
+      }
```

Comment:
We probably shouldn't even be in this function if we failed to find a suitable NDT server. If we're here and `this._hostname` is null, it means the calling code is wrong, so suggest re-wording to something like "No server hostname specified. Aborting."

Also, this is a style preference, so it's up to you, but I generally find code more readable when it's structured like:

```
if (!this._hostname) {
  console.log('whatever');
  return false;
}
this.client = new NDTjs(this._hostname, this._port, this._path,
                        this.callbacks, this._update_interval);
this.client.startTest();
```